src/modules/strategy/strategy_manager.js

```diff
@@ -224,7 +224,7 @@ module.exports = class StrategyManager {
         // check if candle to close time is outside our allow time window
         if (
           validateLookbacks &&
-          !this.technicalAnalysisValidator.isValidCandleStickLookback(lookbacks[exchange].slice(), period)
+          !this.technicalAnalysisValidator.isValidCandleStickLookback(lookbacks[exchange].slice(), period, this.logger)
```

Comment:
Well... I'm not up to date with made architectural decisions and whatnot but generally it's a bad idea to pass logger instance in such fashion. Adds way too much noise.